app/Http/Controllers/Twill/ExhibitionController.php

```diff
@@ -2,90 +2,74 @@
 
 namespace App\Http\Controllers\Twill;
 
-use A17\Twill\Http\Controllers\Admin\ModuleController;
+use A17\Twill\Services\Listings\Columns\Text;
+use A17\Twill\Services\Listings\TableColumns;
+use App\Models\Api\Exhibition;
 use App\Repositories\Api\ExhibitionRepository;
 use App\Repositories\SiteTagRepository;
-use Illuminate\Contracts\Foundation\Application;
-use Illuminate\Http\Request;
 
-class ExhibitionController extends \App\Http\Controllers\Twill\BaseApiController
+class ExhibitionController extends BaseApiController
 {
-    protected $moduleName = 'exhibitions';
-    protected $hasAugmentedModel = true;
-    protected $previewView = 'site.exhibitionDetail';
-
-    protected $indexOptions = [
-        'publish' => true,
-        'bulkEdit' => false,
-        'create' => false,
-        'permalink' => true,
-    ];
-
-    protected $indexColumns = [
-        'image' => [
-            'thumb' => true,
-            'optional' => false,
-            'variant' => [
-                'role' => 'hero',
-                'crop' => 'default',
-            ],
-        ],
-        'title' => [
-            'title' => 'Title',
-            'field' => 'title',
-            'sort' => true,
-        ],
-        'augmented' => [
-            'title' => 'Augmented?',
-            'field' => 'augmented',
-            'present' => true,
-        ],
-        'date' => [
-            'title' => 'Dates',
-            'field' => 'date',
-            'present' => true,
-            'optional' => false,
-            'sort' => true,
-            'sortKey' => 'aic_start_at',
-        ],
-    ];
-
-    protected $indexWith = ['medias'];
-
-    protected $formWith = ['revisions', 'siteTags'];
-
-    protected $defaultOrders;
-
-    protected $filters = [];
-
-    public function __construct(Application $app, Request $request)
+    public function setUpController(): void
     {
-        parent::__construct(...func_get_args());
-
-        $filters = $this->getRequestFilters();
-
-        if (is_array($filters) && ($filters['search'] ?? false)) {
-            $this->defaultOrders = ['_score' => 'desc'];
-        } else {
-            $this->defaultOrders = ['aic_start_at' => 'desc'];
-        }
+        $this->disableBulkDelete();
+        $this->disableBulkEdit();
+        $this->disableCreate();
+        $this->disableDelete();
+        $this->disableEdit();
+        $this->disableRestore();
+        $this->eagerLoadFormRelations(['revisions', 'siteTags']);
+        // I believe this was meant to load local images (as opposed to from the
+        // api), but this no longer seems to work.
+        $this->eagerLoadListingRelations(['medias']);
```

Comment:
This line throws the error for me: 

`Aic\Hub\Foundation\Library\Api\Builders\ApiModelBuilder::eagerLoadRelation(): Argument #1 ($models) must be of type array, Aic\Hub\Foundation\Library\Api\Models\ApiCollection given, called in /home/vagrant/website/vendor/aic/data-hub-foundation/src/Library/Api/Builders/ApiModelBuilder.php on line 464
`

Commenting out `$this->eagerLoadListingRelations(['medias']);` displays the page, but the URL references to the images are missing the UUID: 

```
https://www.artic.edu/iiif/2//full/!80,80/0/default.jpg
```

In between the two slashes after `iiif/2` should be an `image_id` from the API.

Reply:
I've removed the eager load listing relations and enable show image settings to alleviate these immediate problems. I'll create Jira tickets for these two issues.